### PR TITLE
fix:RDCC-2632 added hikari configuration

### PR DESCRIPTION
**Pull request by shreedhar-hmcts**
### JIRA link (if applicable) ###
fix:RDCC-2632 added hikari configuration

### Change description ###
fix:RDCC-2632 added hikari configuration

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -38,6 +38,13 @@ spring:
     password: ${POSTGRES_PASSWORD:dbuserprofile}
     properties:
       charSet: UTF-8
+    hikari:
+      minimumIdle: 2
+      maximumPoolSize: 10
+      idleTimeout: 600000
+      poolName: LRDHikariCP
```
Comment (nitishsingla1234) [inline src/main/resources/application.yaml:45]: Pool name is LRDHikari. Shouldn't it be UPHikariCP?

**Review by nitishsingla1234 — CHANGES_REQUESTED** [review]

**Review by nitishsingla1234 — APPROVED** [review]

**Review by pksaxena01 — APPROVED** [review]
Approved

**Review by AdilOoze — APPROVED** [review]

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -38,6 +38,13 @@ spring:
     password: ${POSTGRES_PASSWORD:dbuserprofile}
     properties:
       charSet: UTF-8
+    hikari:
+      minimumIdle: 2
+      maximumPoolSize: 10
+      idleTimeout: 600000
+      poolName: UPHikariCP
+      maxLifetime: 1800000
+      connectionTimeout: 30000
```
Comment (shashank-rastogi) [inline src/main/resources/application.yaml:47]: maxlifetime seems to be too high

Comment (shreedhar-hmcts) [reply src/main/resources/application.yaml:47]: other reform teams has kept maxlifetime as 60 secs , as compared to them we have kept 30 secs

File: src/main/resources/application.yaml (hunk 1 above)
Comment (shreedhar-hmcts) [reply src/main/resources/application.yaml]: yes renamed

**Review by shashank-rastogi — APPROVED** [review]

**Pull request merged**